UnitTest/tests/test_firmware_download_1-definition.txt

```diff
@@ -16,11 +16,11 @@ defmod test_firmware_download_1 UnitTest dummyDuino
 	
 	subtest 'flash with esp hardware parameter set' => sub {
```

Comment:
Die Bezeichnung des Tests sollten wir in esp8266 ändern und einen 
2.
'flash with esp8266cc1101 hardware parameter set' 
hinterlegen :)

Reply:
Was die Bezeichnungen angeht, so sollte man sich die ALLE mal anschauen :-)

Reply:
:)